crates/store/re_sorbet/src/row_id_column_descriptor.rs

```diff
@@ -34,6 +34,12 @@ impl RowIdColumnDescriptor {
         Self {}
     }
 
+    #[inline]
+    #[expect(clippy::unused_self)]
+    pub fn name(&self) -> &'static str {
```

Comment:
What is the intentional use for this? If this is supposed to be the default column name, then I think we should use either PascalCase or snake_case. If this is supposed to be a human readable string, then I think "Row ID" is better.

Reply:
Human readable name, so "Row ID" it is.